environments/gcp/flyte-core/ingress.tf

```diff
@@ -33,10 +33,11 @@ data kubectl_file_documents "cert-manager-manifest" {
 resource kubectl_manifest "cert-manager-crds" {
```

Comment:
Nit pick:
probably not needed for the CRDs themselves but only the actual certificate later. Doesn't do any harm though.